test/evergreen.yml

```diff
@@ -1638,6 +1662,105 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: check-test-long-1
+    tags: ["check-long-test"]
+    commands: 
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: ac_cv_func_ftruncate=no
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+            ${test_env_vars|} $(pwd)/../test/csuite/random_abort/smoke.sh 2>&1
+            ${test_env_vars|} $(pwd)/../test/csuite/timestamp_abort/smoke.sh 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/test_truncated_log 2>&1
+
+  - name: check-test-long-2
+    tags: ["check-long-test"]
+    commands:
+      - func: "get project"
+      - func: "configure wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-g -Werror"
+          posix_configure_flags: --enable-silent-rules --enable-diagnostic --disable-static
+      - func: "make wiredtiger"
+        vars:
+          make_command: make all
+      - func: "make check all"
+        vars:
+          test_env_vars: TESTUTIL_ENABLE_LONG_TESTS=1
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            WT3363_CHECKPOINT_OP_RACES=1 $(pwd)/test/csuite/./test_wt3363_checkpoint_op_races
+      - func: "many dbs test"
+      - func: "many dbs test"
+        vars:
+          many_db_args: -I
+      - func: "many dbs test"
+        vars:
+          many_db_args: -D 40
+      - func: "many dbs test"
+        vars:
+          many_db_args: -I -D 40
```

Comment:
Those comments in the Jenkins job are useful info that I'd hope to retain during migration.

Reply:
Yes I now realize it will be a good practice to bring in the original comments given some test are very tricky to understand.